src/main.rs

```diff
@@ -207,64 +218,47 @@ fn template(inner: impl Renderable) -> impl Renderable {
-                        container: {
-                            center: true,
-                        },
-                        fontFamily: {
-                            mono: courier, monospace,
-                        }
-                    }
-                }"#
-            </script>
 
-            <meta charset="utf-8"/>
-            <meta name="description" content="Unpacking life with Tris & Robin."/>
-
-            <meta content="width=device-width, initial-scale=1" name="viewport"/>
-            <title class="text-4xl" >"Decapsulate Podcast"</title>
+                <meta charset="utf-8">
+                <meta name="viewport" content="width=device-width, initial-scale=1">
+                <meta name="color-scheme" content="light dark">
+                <meta http-equiv="x-clacks-overhead" content="GNU Terry Pratchett" />
+                <link
+                rel="stylesheet"
+                href="https://cdn.jsdelivr.net/npm/@picocss/pico@2/css/pico.min.css"
+                >
 
             </head>
 
-                <body class="bg-neutral-900 text-white font-mono text-sm md:text-2xl mx-auto w-full">
-
-                    <nav class="bg-neutral-800 flex items-center justify-between flex-wrap p-6">
-                        <div class="flex items-center flex-shrink-0 text-white mr-6">
-                            <span class="font-semibold text-xl tracking-tight"><a href="/">Decapsulate Podcast</a></span>
-                        </div>
-                        <div class="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
-                            <div class="text-xl lg:flex-grow">
-                                // <a href="index.html#about" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
-                                //     About
-                                // </a>
-                                // <a href="" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
-                                //     Listen
-                                // </a>
-                                // <a href="" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
-                                //     Credits
-                                // </a>
-                                <a href="/decapsulate-mp3.xml" class="underline block lg:inline-block lg:mt-0 text-black-200 hover:text-white mr-4">
-                                   Podcast Feed
+            <body>
+                <main class="container">
+                <header>
+                    <nav>
+                        <ul>
+                            <li><a href="/">Decapsulate Podcast</a></li>
+                        </ul>
+                        <ul>
+                            <li>
+                                <a href="/decapsulate-mp3.xml">
+                                    Podcast Feed
                                 </a>
```

Comment:
Browsers can render whitespace around text weirdly. Best to avoid it. 


```suggestion
                                <a href="/decapsulate-mp3.xml">Podcast Feed</a>
```